diff --git a/node_modules/@types/jsonld/jsonld.d.ts b/node_modules/@types/jsonld/jsonld.d.ts
index 5c7cef9..3d39633 100644
--- a/node_modules/@types/jsonld/jsonld.d.ts
+++ b/node_modules/@types/jsonld/jsonld.d.ts
@@ -231,7 +231,7 @@ type Keyword = {
     "@prefix": boolean;
     "@propagate": boolean;
     "@protected": boolean;
-    "@reverse": string;
+    "@reverse": OrArray<string | NodeObject>;
     "@set": OrArray<null | boolean | number | string | NodeObject | ValueObject>;
     "@type": string;
     "@value": null | boolean | number | string;
